roles/ceph-common/tasks/checks/check_mandatory_vars.yml

```diff
@@ -23,26 +27,36 @@
     ceph_stable_rh_storage and
     not ceph_stable_rh_storage_cdn_install and
     not ceph_stable_rh_storage_iso_install
+  tags:
+    - package-install
 
 - name: make sure journal_size configured
   fail:
     msg: "journal_size must be configured. See http://ceph.com/docs/master/rados/configuration/osd-config-ref/"
-  when: journal_size|int == 0
+  when:
+    journal_size|int == 0 and
+    osd_group_name in group_names
```

Comment:
Ok I see.
Yes I'm referring to the ansible handler.

No there is no real issue with having an inconsistent `ceph.conf` expect the inconsistency ;).
Let me run this one time locally but this should be fine in the end.